src/code_8006E9C0.c

```diff
@@ -65,24 +65,24 @@ void reset_object_variable(void) {
 void func_8006EB10(void) {
     s32 i;
     
-    for (i = 0; i < 128; i++) {
-        D_8018C1B0[i] = -1;
+    for (i = 0; i < gObjectParticle1_SIZE; i++) {
```

Comment:
defines should be all caps.

But if you intended this as more like a temporary name because you couldn't think of a better one, then this is fine.

Reply:
it's just a replace of name who also replace const so since the const are link to the variable gObjectParticle keep this name it's fine but if you want we can replace by OBJECT_PARTICLE_X_SIZE if you think it's better